File: sem-2/week-4/week-4-prob-4.cpp
// 25MCD005 WEEK-4 PROBLEM-D Construct the String
#include <iostream>
#include <string>

using namespace std;

int main() {
    int t;
    if (cin >> t) {
        while (t--) {
            int n, a, b;
            cin >> n >> a >> b;
            string s = "";
            for (int i = 0; i < n; i++) {
                s += (char)('a' + (i % b));
            }
            cout << s << endl;
        }
    }
    return 0;
}

File: sem-1/week-4/week-4-prob-4.py
# 25MCD005 SOHAM DAVE WEEK-4 PROBLEM-4 "Increasing Array"
n = int(input())
arr = list(map(int, input().split()))
moves = 0
for i in range(1, n):
    if arr[i] < arr[i - 1]:
        moves += arr[i - 1] - arr[i]
        arr[i] = arr[i - 1]
print(moves)
